src/main/scala/com/yotpo/metorikku/utils/HudiUtils.scala

```diff
@@ -0,0 +1,30 @@
+package com.yotpo.metorikku.utils
+
+import org.apache.hudi.avro.model.HoodieCompactionPlan
+import org.apache.hudi.common.table.{HoodieTableMetaClient, HoodieTimeline}
+import org.apache.hudi.common.table.timeline.HoodieInstant
+import org.apache.hudi.common.table.timeline.HoodieInstant.State
+import org.apache.hudi.common.util.CompactionUtils
+import org.apache.hudi.common.util.collection.ImmutablePair
+import org.apache.hudi.config.HoodieWriteConfig
+import org.apache.hudi.table.HoodieTable
+import org.apache.spark.SparkContext
+import org.apache.spark.api.java.JavaSparkContext
+
+object HudiUtils {
+  def deletePendingCompactions(sparkContext: SparkContext, basePath: String): Unit = {
+    val jsc = JavaSparkContext.fromSparkContext(sparkContext)
+    val hudiMetaclient = new HoodieTableMetaClient(sparkContext.hadoopConfiguration, basePath)
+    val writerConfig = HoodieWriteConfig.newBuilder().withPath(basePath).build()
+    val hudiTable = HoodieTable.getHoodieTable(hudiMetaclient, writerConfig, jsc)
+    val pendingCompactionPlans = CompactionUtils.getAllPendingCompactionPlans(hudiMetaclient)
+    val activeTimeline = hudiTable.getActiveTimeline()
+
+    pendingCompactionPlans.toArray().foreach( { pendingCompactionPlan => {
+      val inflightInstant = pendingCompactionPlan.asInstanceOf[ImmutablePair[HoodieInstant, HoodieCompactionPlan]].getLeft
+      activeTimeline.deleteInflight(inflightInstant)
```

Comment:
@liebstein I think u should also log that u actually deleted something that was pending..
So we could actually know how many times that happend and if it really saved us.